openwisp_users/migrations/__init__.py

```diff
@@ -111,3 +111,26 @@ def update_admins_permissions(apps, schema_editor):
         admin.permissions.add(*permissions)
     except ObjectDoesNotExist:
         pass
+
+
+def create_organization_owners(apps, schema_editor):
+    org_owner_model_name = swapper.get_model_name('openwisp_users', 'OrganizationOwner')
+    org_owner_label = swapper.split(org_owner_model_name)[0]
+    org_owner_model = apps.get_model(org_owner_label, 'OrganizationOwner')
+    org_user_model_name = swapper.get_model_name('openwisp_users', 'OrganizationUser')
+    org_user_label = swapper.split(org_user_model_name)[0]
+    org_user_model = apps.get_model(org_user_label, 'OrganizationUser')
+    org_model_name = swapper.get_model_name('openwisp_users', 'organization')
+    org_model_label = swapper.split(org_model_name)[0]
+    org_model = apps.get_model(org_model_label, 'organization')
```

Comment:
why don't you make a helper method that retrieves the model? So we avoid repeating `get_model_name`, `split` and `get_model` so many times, then you can simply do

```python
OrganizationOwner = get_model('OrganizationOwner')
```
and so on